test/e2e/specs/specs-playwright/wp-support__showme-where.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * @group calypso-pr
+ */
+
```

Comment:
What do you think of adding these as just another describe block under the other Support Popover spec? They seem to fit well within that purview, and it might be tidier to have all those specs in one place!